pkg/cloud/cloud_test.go

```diff
@@ -67,6 +68,10 @@ func TestGetResources(t *testing.T) {
 		platform:     configv1.AzurePlatformType,
 		isAzureStack: true,
 		expected:     azurestack.GetResources(),
+	}, {
+		name:     "AlibabaCloud resources are empty",
```

Comment:
This test description isn't correct
```suggestion
		name:     "AlibabaCloud resources returned as expected",
```